fastlane_core/spec/project_spec.rb

```diff
@@ -430,18 +430,20 @@ def count_processes(text)
         text = "FOOBAR" # random text
         count = count_processes(text)
         cmd = "ruby -e 'sleep 3; puts \"#{text}\"'"
-        # this doesn't work
```

Comment:
What a great comment we had in here, haha

Reply:
it's not wrong...